addons/binding/org.openhab.binding.airvisualnode/src/main/java/org/openhab/binding/airvisualnode/AirVisualNodeBindingConstants.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airvisualnode;
+
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
We are not using `@NonNull` but `@NonNullByDefault` on class level, see https://www.eclipse.org/smarthome/documentation/development/conventions.html